frontend/src/scenes/insights/ComputationTimeWithRefresh.tsx

```diff
@@ -1,14 +1,11 @@
-import { Button } from 'antd'
```

Comment:
There's also `frontend/src/queries/nodes/InsightViz/ComputationTimeWithRefresh.tsx` for data exploration, if we're planning on running the experiment for longer.

Reply:
Ah yes I knew there was something I meant to mention in the description and forgot! I've ignored the data exploration side for now but will make sure changes are implemented there once we settle on the specifics